youtube_dl/extractor/nbc.py

```diff
@@ -15,7 +15,7 @@
 
 
 class NBCIE(AdobePassIE):
-    _VALID_URL = r'https?(?P<permalink>://(?:www\.)?nbc\.com/[^/]+/video/[^/]+/(?P<id>n?\d+))'
+    _VALID_URL = r'https?(?P<permalink>://(?:www\.)?nbc\.com/(?:classic-tv/|)[^/]+/video/[^/]+/(?P<id>n?\d+))'
```

Comment:
1. If you want to make it optional you should use `?` qualifier.
2. Add a test.